tests/examples/examples_smoke_test.go

```diff
@@ -1,10 +1,24 @@
-// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
-//go:build !smoke
```

Comment:
I was looking through the codebase and I see that we removed the flag here, but didn't add it to the examples_test. Is this intentional? @AaronFriel

Reply:
Yeah this looks like a mistake. The `_test.go` should have `//go:build !smoke`